startup/00-base.py

```diff
@@ -54,53 +68,43 @@ def wait_for_connection(self, timeout=DEFAULT_CONNECTION_TIMEOUT):
     timeout = 10  # seconds
     going = "NOT going"
 
-print(f'\nEpicsSignalBase timeout is {timeout} [seconds]. {going} to touch beamline hardware.\n')
+print(f"\nEpicsSignalBase timeout is {timeout} [seconds]. {going} to touch beamline hardware.\n")
 
-from ophyd.signal import EpicsSignalBase
 # EpicsSignalBase.set_default_timeout(timeout=timeout, connection_timeout=timeout)  # old style
 EpicsSignalBase.set_defaults(timeout=timeout, connection_timeout=timeout)  # new style
 
-import datetime
 
 def print_now():
```

Comment:
Looks like we have this `print_now` function defined twice. Need to keep just once copy.